vagrant-service-manager.gemspec

```diff
@@ -4,8 +4,10 @@ require 'vagrant-service-manager/version'
 Gem::Specification.new do |spec|
   spec.name          = 'vagrant-service-manager'
   spec.version       = Vagrant::ServiceManager::VERSION
+  spec.license       = 'GPL-2.0'
   spec.homepage      = 'https://github.com/projectatomic/vagrant-service-manager'
   spec.summary       = "To provide the user a CLI to configure the ADB/CDK for different use cases and to provide glue between ADB/CDK and the user's developer environment."
+  spec.description   = 'Enables various Linux container development environments in ADB/CDK'
```

Comment:
how about adding following in `spec.description`
"Provides setup information, including environment variables and certificates, required to access services provided by an Atomic Developer Bundle (ADB). This plugin makes it easier to use the ADB with host-based tools such as Eclipse and docker, kubernetes, OpenShift  CLI commands."